React/Modules/RCTUIManager.m

```diff
@@ -1222,6 +1222,33 @@ - (void)setNeedsLayout
   }];
 }
 
+static void RCTViewIsAncestorOf(RCTShadowView *view,
+                                RCTShadowView *ancestor,
+                                RCTResponseSenderBlock callback)
+{
+  if (!view) {
+    return;
+  }
+  if (!ancestor) {
+    return;
+  }
```

Comment:
That’s totally fair. I’m just providing some general code feedback, following existing styles is definitely a good choice 👍